src/provider/application.js

```diff
@@ -72,7 +73,18 @@ class Application extends EventEmitter {
       const width = calculateWidth(this.resizeConfig.WidthCalculationMethod);
       this.JSONRPC.notification('resize', [null, `${width}px`]);
     } else {
-      const height = calculateHeight(this.resizeConfig.heightCalculationMethod);
+      let height = calculateHeight(this.resizeConfig.heightCalculationMethod);
+
+      if (this.targetNodes.length > 0) {
+        const maxTargetHeight = this.targetNodes
+          .map(selector => document.querySelector(selector))
```

Comment:
shouldn't this be querySelectorAll?

Reply:
This is arguable, but I feel we should limit the usage of querySelectorAll. When the user provide a css-selector, they should give a selector that can uniquely identify a dom node instead of a couple. Reason:
1. performance wise querySelectorAll is worse.
2. If user want to select a list of sibling nodes, they can actually provide the selector of the parent node of those sibling nodes.
3. If the selector selects a list of nodes that are not siblings, the page creator should now which one is the last and they should provide the selector to the last node instead.

So in above scenarios, I think we should not generalize this use case. The feature of the target nodes itself is a special case and we probably want the user to pay more attention to it when using it.

Reply:
The scenario i'm thinking of is on  a workflow where there are dropdowns.. Each doesn't have a specific selector, I just know I always want dropdowns to be accounted for. Its easier and safe to just scan for them rather than ensure each one has a specific selector.